release/nightly_tests/dataset/read_tfrecords_benchmark_app.yaml

```diff
@@ -0,0 +1,14 @@
+base_image: {{ env["RAY_IMAGE_ML_NIGHTLY_GPU"] | default("anyscale/ray-ml:nightly-py37-gpu") }}
```

Comment:
This is copied from `app_config.yaml`. The only change is line 13 to install crc32c package.